Jetcaster/wear/src/main/java/com/example/jetcaster/ui/JetcasterNavController.kt

```diff
@@ -45,6 +52,18 @@ public object LatestEpisodes : NavigationScreens("latestEpisodes") {
     public fun destination(): String = navRoute
 }
 
+public object PodcastDetails : NavigationScreens("podcast?podcastUri={podcastUri}") {
+    public const val podcastUri: String = "podcastUri"
+    public fun destination(podcastUri: String): String = "podcast?podcastUri=$podcastUri"
```

Comment:
It's worth checking whether you need to encode this. You might be skipping this https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:navigation/navigation-common/src/main/java/androidx/navigation/NavType.kt;l=668

Reply:
added encoding and decoding